src/sagemaker/session.py

```diff
@@ -643,6 +643,7 @@ def create_model_from_job(self, training_job_name, name=None, role=None, primary
             str: The name of the created ``Model``.
         """
         training_job = self.sagemaker_client.describe_training_job(TrainingJobName=training_job_name)
+        tags = self.sagemaker_client.list_tags(ResourceArn=training_job['TrainingJobArn'])['Tags']
```

Comment:
I'm actually not sure whether or not the tags from the training job should propagate to the SageMaker model  as well.

What are your reasons for that?

Reply:
Hi @ChoiByungWook 
Since the method creates a Model from the training job, i thought fetching the tags details from the training job would make sense.

Reply:
Makes sense.

I think it is fair to fetch the tags from the training job and attach to the model as well.

Do you think there are any cases in which this can be undesirable? (Just want to cover bases and get opinions.)

Reply:
ok! so,
when using the `sagemaker.estimator.Estimator` 's `transformer`, the model will be created with the tags provided in the estimator's constructor and not with the tags in the transformer method. 

whereas the model created from any framework's estimator will take tags from the transformer method. 

is this an acceptable discrepancy? 
If not, and if we should take the model tags from the transformer method, we'll have to update the `Session.create_model_from_job` method to accept tags as argument.

This will also handle the use cases where a user would want to use the create_model_job method directly and provide their custom tags for the model.

Reply:
I think it makes more sense to add a tags parameter to the create_model_from_job method. It is only being used in the `transformer` function only as of now, so I think it should utilize the tags passed when calling transformer.

I think I want to follow the same logic as the deploy calls, where tags from the deploy call are propagated to the model, endpoint config and endpoint.

You can also do both and have it default to using tags from the training job if no tags are provided, up to you.

Thanks!